.github/workflows/build_dist.yml

```
@@ -48,14 +48,14 @@ jobs:
           - macos-13 # x86
```

```suggestion
          - macos-15-intel # x86
```
https://github.blog/changelog/2025-09-19-github-actions-macos-13-runner-image-is-closing-down